### DISCUSSION TITLE
Repository page can't be opened. Error code 500 (Looks like something went wrong!)

### DISCUSSION THREAD

**Discussion by Quviry** [community/community#40225] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've faced a problem with my [team repo](https://github.com/masters-of-all-handlers/uservice-dynconf). After a new pull we've start to get error 500 on opening repo page. It's happens not every time. but with frequency over then 50%.

![image](https://user-images.githubusercontent.com/55559092/204160990-aee2fbff-ffc1-4c38-8b42-9248b4a18209.png)
***
Some other parts of commits data is unavailable too.

![image](https://user-images.githubusercontent.com/55559092/204160909-533a868b-b153-4cdb-adbd-9af9cb6edc95.png)
***

This is the result of accidentally CI/CD test. Luckily it was caused with very small change (1 line) that can't be seen on picture above.

 ![image](https://user-images.githubusercontent.com/55559092/204160948-99af91e6-4f38-4d69-b7bc-ecb2f9c2183b.png)

**Comment by ossanna16** [comment]
Hi there @Quviry and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
